README.md

```diff
@@ -11,25 +11,32 @@ Your personal practice companion
 </p>
 
 <br/>
+
+<p align="center">
+<a href='https://play.google.com/store/apps/details?id=app.musikus&hl=de&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'>
```

Comment:
I don't think, we should use the GooglePlay Badge while the link only works for our alpha testers.

Reply:
Do you think it confuses people who don't read the description and wonder why the link is not working 😅?